src/HotChocolate/Stitching/src/Stitching/Utilities/HttpQueryClient.cs

```diff
@@ -235,13 +235,11 @@ await httpClient.PostAsync(default(Uri), requestContent)
                         break;
 
                     case IntValueNode i:
-                        jsonWriter.Flush();
-                        WriteNumberValue(writer, i.Value);
+                        jsonWriter.WriteNumberValue(Convert.ToInt32(i.Value));
```

Comment:
I think this should be Int64

Reply:
The way it is implemented is not good... I think I need to rework this further ... the convert is exactly what I did not want here. Rather better would be just to write the string representation that is already there. Will look at this, this weekend.

Reply:
https://github.com/ChilliCream/hotchocolate/pull/2086 is something like that better ?